### PR TITLE
Design choice

### PR DESCRIPTION
**Pull request by Ichikamp**

### CODE REVIEW

File: Report/sections/2_design/Design-Choice [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+\section{Design choice}
+
+
+The language is intended to be a transitional language from visual programming to high-level language. Based on the problem analysis, it has been possible to specify the design for the user so that it hits the target group.
+ The target group is beginners in programming who are younger people. Therefore a programming language to make computer games has been built. This has been chosen as young people can relate more to video games as their hobby.
```
Comment (johndensaj) [inline Report/sections/2_design/Design-Choice:6]: Kunne godt uddybes en del:
"This has been chosen as young people can relate more to video games as their hobby" -  skrev vi ikke lidt om det her i starten? måske lidt at hente derfra. 
Men i hvert fald en kilde på at understøtte påstanden.

File: Report/sections/2_design/Design-Choice [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+\section{Design choice}
+
+
+The language is intended to be a transitional language from visual programming to high-level language. Based on the problem analysis, it has been possible to specify the design for the user so that it hits the target group.
+ The target group is beginners in programming who are younger people. Therefore a programming language to make computer games has been built. This has been chosen as young people can relate more to video games as their hobby. 
+ This will help them to understand the principles of how a game should work more easily, and therefore be able to start programming.
```
Comment (johndensaj) [inline Report/sections/2_design/Design-Choice:7]: Maybe give an example of what principles?

File: Report/sections/2_design/Design-Choice [hunk 3]
```diff
@@ -0,0 +1,10 @@
+
+\section{Design choice}
+
+
+The language is intended to be a transitional language from visual programming to high-level language. Based on the problem analysis, it has been possible to specify the design for the user so that it hits the target group.
+ The target group is beginners in programming who are younger people. Therefore a programming language to make computer games has been built. This has been chosen as young people can relate more to video games as their hobby. 
+ This will help them to understand the principles of how a game should work more easily, and therefore be able to start programming. 
+ It was decided to develop a language inspired on the game "the Legend of Zelda: Link’s Awakening" into the programming language, as it is a familiar game with basic game terminologies.
```
Comment (johndensaj) [inline Report/sections/2_design/Design-Choice]:
```suggestion
 It was decided to develop a language inspired by the game "the Legend of Zelda: Link’s Awakening" into the programming language, as it is a familiar game with basic game terminologies.
```

Comment (johndensaj) [inline Report/sections/2_design/Design-Choice]: "as it is a familiar game with basic game terminologies" 

måske ændre 'familiar' med 'well known'?

File: Report/sections/2_design/Design-Choice [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+  This helps to set framework for what the program language should be capable of.
```
Comment (johndensaj) [inline Report/sections/2_design/Design-Choice]:
```suggestion
  This helps to set a framework for what the program language should be capable of.
```

File: Report/sections/2_design/Design-Choice [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+ In addition, "Unity" has been used as a program to test different functions to find out what could work well for the program language.
```
Comment (johndensaj) [inline Report/sections/2_design/Design-Choice]: Synes det kommer lidt ud af ingenting.

**Review by johndensaj — COMMENTED** [review]
.

**Review by chhoumann — APPROVED** [review]

**Pull request merged**